Fragen/Kap.8.adoc

```diff
@@ -318,7 +343,7 @@ endif::solution[]
 ### Frage
 [source,java]
 ----
-boolean isOdd(int n) {
+boolean isEven(int n) {
```

Comment:
Da haben Sie leider einen Fehler mit einem Fehler korrigiert ;-) `isEven` muss auch in der Rekursion vorkommen. Habe im Master `isOdd` korrigiert zu:

~~~
boolean isOdd(int n) {
  return n >= 0 && (n == 1 || isOdd(n-2));
}
~~~